enhancements/installer/packet-ipi.md

```diff
@@ -0,0 +1,197 @@
+  Official images can be hosted outside of Packet, such as ContainerLinux and RancherOS. More details area available here: <https://github.com/packethost/packet-images>
+  Custom Images can be hosted on git, as described here:
+  - <https://www.packet.com/developers/docs/servers/operating-systems/custom-images/>
+  - <https://www.packet.com/developers/changelog/custom-image-handling/>
+- Packet does not offer a managed DNS solution. Route53, CloudFlare, or RFC 2136
+  (DNS Update), and NS record based zone forwarding are considerations. Existing
+  IPI drivers like BareMetal have similar limitations.
+- LoadBalancing can be achieved with the Packet CCM, which creates and maintains
+  a MetalLB deployment within the cluster.
+- Both Public and Private IPv4 and IPv6 addresses made available for each node.
+  Private and VLAN Networks can be restricted at the project level (default) or
+  node by node. While custom ASNs and BGP routing features are supported, these
+  and other networking features will not be exposed through the IPI.
+  <https://www.packet.com/cloud/network/>
+- Openshift with run directly on RHCOS directly on baremetal without need for a
+  VM layer.
+
+### Risks and Mitigations
+
+The official Packet ClusterAPI, CCM, and CSI drivers are all relatively new and
+may make undertake substantial design changes.  This project will either need to
+adopt and maintain the current releases, or adopt newer releases developed
+alongside this project.
+
+The CI will need to assure that Packet resources are cleanly released after failed and successful builds.  Packet is currently used in the baremetal provisioning tests, so there is some prior art available.  If the same account is used for testing the IPI, it may need account quota increases.
+
+## Design Details
+
+### Test Plan
+
+We will use existing UPI platforms, such as OpenStack, AWS, and GCP, as the
+inspiration for our testing strategy:
+
+- A new e2e job will be created
+- At the moment, we think unit tests will probably not be necessary. However, we
+  will cover any required changes to the existing codebase with appropriate
+  tests.
+
+### Graduation Criteria
+
+The proposal is to follow a graduation process based on the existence of a CI
+running suite with end-to-end jobs. We will evaluate its feedback along with
+feedback from QE's and testers.
+
+We consider the following as part of the necessary steps:
+
+- CI jobs present and regularly scheduled.
+- IPI document published in the OpenShift repo.
+- End to end jobs are stable and passing and evaluated with the same criteria as
+  comparable IPI providers.
+- Developers of the team have successfully used the IPI to deploy on Packet
+  following the documented procedure.
+
+## Implementation History
+
+Significant milestones in the life cycle of a proposal should be tracked in
+`Implementation History`.
+
+## Drawbacks
+
+The IPI implementation is provisioned on baremetal which faces resource
```

Comment:
```suggestion
The IPI implementation is provisioned on bare metal which faces resource
```

I imagine the Packet team may have strong opinions on this matter, ha!  This is a total nit, but we have a "bare metal" style guide that we follow here: https://github.com/metal3-io/metal3-docs/blob/master/design/bare-metal-style-guide.md

Reply:
Packet's standard agrees. I have to get it into my head. 🤕 🔨